api-js/src/queries/domains/find-my-domains.js

```diff
@@ -7,7 +7,31 @@ const findMyDomains = {
   args: {
     ...connectionArgs,
   },
-  resolve: async () => {},
+  resolve: async (
+    _,
+    args,
+    {
+      userId,
+      loaders: {
+        domainLoaderConnectionsByUserId,
+      },
+    },
+  ) => {
+    let domainConnections
+
+    try {
+      domainConnections = await domainLoaderConnectionsByUserId({args})
```

Comment:
Unless you use the spread operator, you don't need to include the curly braces because then you will be wrapping an object inside and object, and the function won't be able to de-structure it properly